dags/oup/parser.py

```diff
@@ -136,3 +141,21 @@ def _get_authors(self, article: ET.Element):
                     }
                 )
         return authors
+
+    def __get_date(self, article):
```

Comment:
I won't make it private because previously you were not using private methods, so let's be consistent